src/main/scala/io/iohk/ethereum/domain/Account.scala

```diff
@@ -1,21 +1,26 @@
 package io.iohk.ethereum.domain
 
 import akka.util.ByteString
-import org.spongycastle.util.encoders.Hex
+
+object Account {
+  val Empty = Account(0, 0, ByteString.empty, ByteString.empty)
+}
 
 case class Account(
   nonce: BigInt,
   balance: BigInt,
   storageRoot: ByteString,
   codeHash: ByteString) {
 
-  override def toString: String = {
-    s"""Account{
-         |nonce: $nonce
-         |balance: $balance wei
-         |storageRoot: ${Hex.toHexString(storageRoot.toArray[Byte])}
-         |codeHash: ${Hex.toHexString(codeHash.toArray[Byte])}
-         |}
-       """.stripMargin
-  }
+  def updateBalance(value: BigInt): Account =
+    copy(balance = balance + value)
+
+  def increaseNonce: Account =
+    copy(nonce = nonce + 1)
+
+  def withCode(codeHash: ByteString): Account =
+    copy(codeHash = codeHash)
```

Comment:
just comment - EVM code is assigned when account is created and it cannot change